src/Atmos/Model/moisture.jl

```diff
@@ -154,16 +145,11 @@ function flux_first_order!(
     moist::EquilMoist,
     atmos::AtmosModel,
     flux::Grad,
-    state::Vars,
-    aux::Vars,
-    t::Real,
-    ts,
-    direction,
+    args,
 )
     tend = Flux{FirstOrder}()
-    args = (atmos, state, aux, t, ts, direction)
     flux.moisture.ρq_tot =
-        Σfluxes(eq_tends(TotalMoisture(), atmos, tend), args...)
+        Σfluxes(eq_tends(TotalMoisture(), atmos, tend), atmos, args)
```

Comment:
I see `atmos` is added to the arguments for `Σfluxes` here, but goes into `args` in sub-components... I sort of see why, but more consistency might be better?

Reply:
It should be consistent (in `precipitation.jl`), thanks!